rules/windows/builtin/security/win_security_access_token_abuse.yml

```diff
@@ -0,0 +1,29 @@
+title: Access Token Abuse
+id: 02f7c9c1-1ae8-4c6a-8add-04693807f92f
+status: experimental
+description: 'This rule tries to detect token impersonation and theft. (Example: DuplicateToken(Ex) and ImpersonateLoggedOnUser with the LOGON32_LOGON_NEW_CREDENTIALS flag.)'
+references:
+    - https://attack.mitre.org/techniques/T1134/001/
```

Comment:
Find these refs : 
 - https://www.elastic.co/fr/blog/how-attackers-abuse-access-token-manipulation 
 - https://www.manageengine.com/log-management/cyber-security/access-token-manipulation.html

Reply:
Thank you for the references. I would be happy to add them.